cumulus_library_opioid/vocab/rxnorm_vsac_builder.py

```diff
@@ -0,0 +1,83 @@
+"""Builder for generating subsets of RxNorm data from a given valueset"""
+import pathlib
+
+from cumulus_library import base_table_builder, base_utils, study_manifest
+from cumulus_library.template_sql import base_templates
+
+from cumulus_library_opioid.vocab import vsac
+
+
+class RxNormVsacBuilder(base_table_builder.BaseTableBuilder):
+    display_text = "Creating RxNorm subsets from VSAC definitions..."
+    base_path = pathlib.Path(__file__).resolve().parent
+
+    def prepare_queries(
+        self,
+        config: base_utils.StudyConfig,
```

Comment:
as I write this more and more - would it be easier/more sensible for other implementers for this dataclass to be in a dedicated module?

Reply:
For public APIs, I usually prefer to hide all submodule details - it allows the flexibility to move things around, submodule layouts for small APIs can often be a needless barrier-to-understanding, and it's clearer what the public API is meant to be.

i.e. I'd do something like this in `cumulus_library/__init__.py` to expose the class at the top:
```
from .base_utils import StudyConfig
```

Then all callers can just grab everything they need from `cumulus_library` as a blessed API, rather than rooting around in the submodules (where who knows whether a thing was intended to be public or not, unless there are clear docs or rigorous use of a sunder prefix).

Code inside `cumulus-library` could keep using the submodule name of course, to help avoid circular imports (or you could solve that another way).

Reply:
i vibe it - i was sort of thinking of this all as 'one big distributed thing' but that's probably easier. Won't do that now, but i'll hit that the next time i'm in here after I add that to the core.